src/test/ui/rfc-2627-raw-dylib/raw-dylib-msvc-only.rs

```diff
@@ -0,0 +1,9 @@
+// only-windows
+// only-gnu
```

Comment:
I think `only` are not addictive and it's shorter to write it like that:
```suggestion
// only-windows-gnu
```